src/Websites/Websites/help/Set-AzWebApp.md

```diff
@@ -24,8 +24,8 @@ Set-AzWebApp [[-AppServicePlan] <String>] [[-DefaultDocuments] <String[]>] [[-Ne
  [-ContainerRegistryUser <String>] [-ContainerRegistryPassword <SecureString>]
  [-EnableContainerContinuousDeployment <Boolean>] [-HostNames <String[]>] [-NumberOfWorkers <Int32>] [-AsJob]
  [-AssignIdentity <Boolean>] [-HttpsOnly <Boolean>] [-AzureStoragePath <WebAppAzureStoragePath[]>]
- [-AlwaysOn <Boolean>] [-MinTlsVersion <String>] [-FtpsState <String>] [-ResourceGroupName] <String>
- [-Name] <String> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+ [-AlwaysOn <Boolean>] [-MinTlsVersion <String>] [-FtpsState <String>] [-HealthCheckPath <String>]
```

Comment:
Don't expose this a parameter directly in the Set-AzWebapp. Please expose this in the  PSSite object this is identified & returned in the response & can be set using the -Webapp property that take a PSSite as input.

Reply:
Yes, updates as per the suggestions.